E2E/pageObjects/game.page.ts

```diff
@@ -133,6 +143,26 @@ export = {
     game.voteIsVisible(chessPiece, tile, value);
   },
 
+  async openPopUp () {
+    I.usePlaywrightTo('open pop-up', async ({ page }) => {
+      await page.evaluate("document.querySelector(`div[data-testid = 'chess-tile-0-6'] > div.pop-over`).classList.add('pop-over-opened')");
+    });
+  },
+
+  checkElementsInThePopUp: (username: string) => {
+    I.waitForElement(game.locator.chessBoard.openedPopUp);
+    I.seeElement(game.locator.chessBoardPopUp.square);
+    I.seeElement(game.locator.chessBoardPopUp.chessPiece);
+    I.seeElement(game.locator.chessBoardPopUp.avatar);
+    I.seeElement(game.locator.chessBoardPopUp.player(username));
+    I.seeElement(game.locator.chessBoardPopUp.score);
+  },
+
+  extractNumbers: async (string: string) => {
+    const regexp = /[0-9]./;
+    return Number(string.match(regexp)[0]);
```

Comment:
Will it be able to extract numbers made of two digits?

Reply:
Yes, I tried it, this function can do that.

Reply:
2nd and third cases should return `123` and `12` respectively

Reply:
I changed it to return only the first number since we only expect one number, and increased possible number size.

Reply:
If this will ever be used on strings like `usern4me 13` it won't work. fix this, please.